requirements/prod.in

```diff
@@ -22,3 +22,4 @@ whitenoise
 google-auth-oauthlib
 marshmallow
 webargs
+webvtt-py
```

Comment:
> Should we be worried that, if they don't support newer Python versions, it's because it is not super well maintained?

WebVTT is a pretty simple format so I think the library was essentially "done" and doesn't need much maintenance. If we do find issues then we can create PRs or create our own parsing code.